addons/l10n_ar_withholding/demo/account_supplier_invoice_demo.xml

```diff
@@ -0,0 +1,33 @@
+<odoo>
+    <record id="l10n_ar_withholding_demo_sup_invoice_1" model="account.move" context="{'allowed_company_ids': [ref('base.company_ri')]}">
```

Comment:
@jco-odoo @vbe-odoo I did in xml because I took similar approach to l10n_ar https://github.com/odoo/odoo/blob/master/addons/l10n_ar/demo/account_supplier_invoice_demo.xml

Reply:
I am a bit surprised that we still have it like that for l10n_ar...  It should not be much work.

Reply:
I moved it to l10n_ar in this pr 
https://github.com/odoo/odoo/pull/179831